scripts/misc/update_meta_coord.pl

```diff
@@ -50,7 +51,7 @@
 my ($variation_dbas, $core_dbas);
 if (defined $opts->{dbname}) {
   $variation_dbas = get_dbas($opts);
-  die "Variation database required" unless ($opts->{dbname} =~ m/variation/);
+  die "Variation database required" unless ($opts->{dbname} =~ m/variation/ || $opts->{type} eq "privates");
```

Comment:
`$opts->{type} eq "privates"`

Don't you mean `private`?

Reply:
Updated